doc/whatsnew.rst

```diff
@@ -10,67 +10,69 @@ A catalog of new features, improvements, and bug-fixes in each release.
 v0.2.8.dev
 ----------
 
-- Correct spectral estimation in :func:`pyriemann.utils.covariance.cross_spectrum` to obtain equivalence with SciPy
+- Correct spectral estimation in :func:`pyriemann.utils.covariance.cross_spectrum` to obtain equivalence with SciPy. :pr:`133` by :user:`qbarthelemy`
```

Comment:
@sylvchev @qbarthelemy @plcrodrigues @gabelstein can you check if I did not screw up the credit assignment of each entry below? 🙏

Reply:
ok !

Reply:
looks good

Reply:
It's ok for me!